tests/headed-cases/blog-lines.js

```diff
@@ -62,6 +62,6 @@ describe('blog-lines example', function () {
     // this permits a large delta to pass on CI.  It visually is rendered
     // correctly, though with seemingly different aliasing choices by the
     // renderer
-    imageTest.imageTest('exampleBlogLines10k', '#map', 0.04, done, null, 1000, 2, '.geojs-map.ready');
+    imageTest.imageTest('exampleBlogLines10k', '#map', 0.04, done, null, 5000, 2, '.geojs-map.ready');
```

Comment:
why this increase to 5000?

Reply:
Travis was timing out on this.  I don't know why, but increasing it allowed the tests to pass.  It is like they are using a different mesa library than before that is slower, but that is just speculation.